recipes/funcannot/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: funcannot
+  version: "v2.8"
+
+source:
+  url: https://github.com/BioTools-Tek/funcannot/archive/master.zip
+  md5: 2e70b503a1632998deba759b50c0c07f
+   
+
+build:
+  number: 0
+  #skip: True # [osx]
+
+requirements:
+  build:
+    - gcc
+    - zlib {{CONDA_ZLIB}}*
```

Comment:
is this really needed or linked in statically? I'm just wondering why it is not needed as runtime dependency.

Reply:
Ah. I thought zlib was needed by conda at build time to unpack the tool. 
So these are includes that are fed to the actual build script, okay I understand -- will remove.